controller/tests/server/test_route_processing.py

```diff
@@ -1382,6 +1357,30 @@ def test_after_request__redacts_recording_folder_path_from_get_recordings_log_me
     }
 
 
+def test_after_request___200_status_code_system_status_log_messages_filtered(
+    client_and_server_manager_and_shared_values, test_client, mocker
+):
+    *_, shared_values_dict = client_and_server_manager_and_shared_values
+    shared_values_dict["system_status"] = CALIBRATED_STATE
+    spied_server_logger = mocker.spy(server.logger, "info")
+    response = test_client.get("/system_status")
+
+    assert response.status_code == 200
+    assert len(spied_server_logger.call_args_list) == 0
+
+
+def test_after_request__status_code_system_status_error_logged(
+    client_and_server_manager_and_shared_values, test_client, mocker
+):
+    *_, shared_values_dict = client_and_server_manager_and_shared_values
+    shared_values_dict["system_status"] = CALIBRATED_STATE
+    spied_server_logger = mocker.spy(server.logger, "info")
+    response = test_client.put("/system_status")
+
+    assert response.status_code != 200
```

Comment:
This test is identical to the one above however you are checking that the status code is not 200. You'll need to actually mock this route or configure the shared values so that this route does not return 200, otherwise this test will always fail

Reply:
The second test makes a PUT call to the server instead of a GET call. This returns an error code that is not 200 and logs `/system_status`.  `spied_server_logger` is then used to check if any logs have been made. When testing `system_status` filtering,  a normal GET request is made, but this time `spied_server_logger` should not return any logs being made.

I can combine these two tests with parametrize.

What would be a some other ways to trigger an error other than changing the request method?